### PR TITLE
New package: owncloud-9.0.2

### PR DESCRIPTION
**Pull request by Gottox**
I've got enough of of updating owncloud by hand. rfc ;)

### CODE REVIEW

File: srcpkgs/owncloud/INSTALL [hunk 1]
```diff
@@ -0,0 +1,10 @@
+case "$ACTION" in
+post)
+	mkdir -p var/db/owncloud/data var/db/owncloud/apps var/log/owncloud
+	chown -R _owncloud var/db/owncloud etc/owncloud
```
Comment (Vaelatern) [inline srcpkgs/owncloud/INSTALL]: Not more with `make_dirs`?

Comment (Gottox) [inline srcpkgs/owncloud/INSTALL]: true, fixed.

**Comment by Vaelatern** [comment]
I like the notion of xbps handling things for webapps. I don't know how much of this needs to be in the tree; I'd prefer patching the default `php.ini` if one is provided.
It's great having the package manager do things for you instead of yourself.

**Comment by Vaelatern** [comment]
@voidlinux/pkg-committers Anybody want to weigh in on if things like webapps should be merged and handled by xbps?

**Comment by Gottox** [comment]
I'll replace this pkg with nextcloud anyway in the next few days.

**Pull request closed without merging**